lighthouse-core/gather/gather-runner.js

```diff
@@ -374,7 +374,9 @@ class GatherRunner {
     return {
       fetchTime: (new Date()).toJSON(),
       LighthouseRunWarnings: [],
-      UserAgent: await options.driver.getUserAgent(),
+      HostUserAgent: await options.driver.getUserAgent(),
+      NetworkUserAgent: '', // updated later
```

Comment:
`NetworkUserAgent` doesn't seem self-obvious to me but I don't have a great suggestion for an alternative. Will brainstorm :)

Reply:
sg, I also tried `RuntimeUserAgent` but thought NetworkUserAgent was a bit more descriptive

Reply:
yeah. I still have nothings, so that's fine with me. If @paulirish has ideas we can do a follow up PR quickly before we ship it :)